bin/beeline

```diff
@@ -23,8 +23,29 @@
 # Enter posix mode for bash
 set -o posix
 
+realpath () {
```

Comment:
I mean the file would be in `/usr/bin` too, yes. I understand it's a little ugly, yeah.
Hm, I wonder if this is really the right approach to finding where Spark is installed. Shouldn't this be covered by `SPARK_HOME`? that's how other software works. The title of the JIRA is actually misleading since symlinks works fine, themselves.